src/ESLint.Formatter/sarif-with-rules.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * @fileoverview SARIF v2.0 formatter
+ * @author Chris Meyer
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Helper Functions
+//------------------------------------------------------------------------------
+
+/**
+ * Returns the severity of warning or error
+ * @param {Object} message message object to examine
+ * @returns {string} severity level
+ * @private
+ */
+function getResultLevel(message) {
+    if (message.fatal || message.severity === 2) {
```

Comment:
>message.severity [](start = 25, length = 16)

what are the full range of severity code and what do they mean?

Reply:
message.fatal is a boolean
severity 1 = warning
severity 2 = error

---
In reply to: [249138554](https://github.com/Microsoft/sarif-sdk/pull/1222#discussion_r249138554) [](ancestors = 249138554)